recipes/exchangelib/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "exchangelib" %}
+{% set version = "3.1.1" %}
+{% set sha256 = "33e76159fa14e004ae8333ae0a0d2bd9c6d4f682f9f63ad659c058ecadb20fef" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [py<35]
```

Comment:
I think this recipe can be `noarch`, so please remove the `skip` line and use `noarch: python` instead.

Reply:
I was wondering about this. There are optional dependencies for Windows: https://github.com/ecederstrand/exchangelib/blob/master/setup.py#L44-L48

What is the best way to handle these?

Reply:
Ah, in that case you can not use `noarch`, since you will be having selectors for Windows, and selectors don't work with `noarch`, https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages. 
Ignore the `noarch` comment, leave the `skip` line, and in `host` add:
```
requests-negotiate-sspi  # [win]
requests-kerberos  # [win]
```

Reply:
Fixed in latest commit. Looks like `requests-kerberos` shouldn't be restricted to Windows.